crates/fluvio-cli-common/Cargo.toml

```
@@ -17,17 +17,20 @@ default = ["fluvio-types"]
 file-records = ["fluvio-protocol/record", "fluvio-protocol/api"]
 version-cmd = ["dep:current_platform", "dep:clap", "dep:sysinfo"]
 smartmodule-test = ["file-records", "dep:fluvio-sc-schema", "dep:fluvio-smartmodule", "dep:fluvio", "dep:fluvio-smartengine", "dep:clap"]
+serde = ["dep:serde", "dep:serde_json"]
 
 [dependencies]
 anyhow = { workspace = true }
 current_platform = { workspace = true, optional = true }
-clap = { workspace = true, optional = true }
+clap = { workspace = true, optional = true, features = ["derive", "std"] }
```

These features are needed to suppress errors when running tests:

```bash
➜  fluvio-cli-common git:(3806-version-support-for-json-output) ✗ cargo test --features serde --features version-cmd creates_json_output
   Compiling clap_builder v4.4.8
   Compiling fluvio-cli-common v0.0.0 (/infinyon/fluvio/crates/fluvio-cli-common)
error: `std` feature is currently required to build `clap`
  --> /Users/esteban/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.4.8/src/lib.rs:32:1
   |
32 | compile_error!("`std` feature is currently required to build `clap`");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```